### PR TITLE
Add flipcard styling to OER listing

### PR DESCRIPTION
**Pull request by gnuj**
This request brings the flipcard styling from the new TYPO3 integration to this GitHub page. It's achived by change the metadata element from a separate list element to a figcaption element inside the OER items and some minor adaptions to the styling and the JS functions.

### CODE REVIEW

**Comment by gnuj** [comment]
@wrznr @herreio Currently im not able to assign reviewers so i have to mention you in this comment. Hope it'll do the job.

**Review by herreio — APPROVED** [review]
Hi @gnuj, this is a nice surprise. I looked at the request and, for my sake, it could be merged. Thanks a lot! Not only for implementing the flip cards but also for ensuring consistent coding styles :)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (wrznr) [inline .editorconfig:1]: Indeed: https://github.com/editorconfig/editorconfig-vim#readme

**Review by wrznr — APPROVED** [review]
Many thanks @gnuj!

**Pull request merged**